examples/integrations/oci/base/identity-se/src/main/resources/application.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2021 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+oci:
+  auth:
+    user: "ocid1.user.oc1.."
```

Comment:
What are these hard-coded values ?

Reply:
Since I cant submit my or someone else's OCI credential, I will just add some text like "your credentials". Please confirm this acceptable.

Reply:
Yes, that's acceptable but don't make them look like actual secret values.

Reply:
Ok, already submitted.